controllers/climatecontroller.py

```diff
@@ -1,15 +1,6 @@
-from TestData import*
-from canBus import*
-from Controller import*
-class ClimateController(Controller):
+class ClimateController():
+  testData
```

Comment:
See the first comment on backupcontroller.py with respect to lines 2 and 3 here.